content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,689 @@
+* Using optional vertex streams
+* Adding passes to your material type
+* Using the Pass Tree Visualizer to debug passes
+
+The VegetationBending materialtype allows materials to bend and sway, simulating how wind affects vegetation. It allows for detail bending with slight movement of branches and leaves, as well as movement of the entire object. 
+
+The code in this tutorial can be found in the [AtomTutorials Gem](https://github.com/o3de/sample-code-gems/tree/main/atom_gems/AtomTutorials). There, you can find the template code to use to follow this tutorial, the final code, and the assets we are using.
+
+As we go along, you may wish to reference the[Material Types and Shaders guide](get-started-materialtypes-and-shaders), which gives higher-level explanations of the mechanisms we are using.
+
+## Get started
+Before you can create your material type, ensure you have [installed the engine](/docs/welcome-guide/setup/), [set up a project](/docs/welcome-guide/create/), and [launched the editor](/docs/welcome-guide/tours/editor-tour).
+
+Next, perform the following steps to get started on making the vegetation bending material type.
+1. Download or clone the Sample Code Gems repository from [here](https://github.com/o3de/sample-code-gems).
+1. The template files that we will be using for this tutorial are in `atom_gems/AtomTutorials/Templates/VegetationBending`. Move all the files in that folder to `{your-project-path}\Materials\Types\`.
+   * You'll notice that there are a couple key files and file extensions that we need to make our custom material type. More details can be found in [this guide](get-started-materialtypes-and-shaders/#1-set-up-the-files), but this tutorial will explain each file as we go along.
+1. Move all the files in `atom_gems/AtomTutorials/Assets/VegetationBending/Objects/` to `{your_project_path}\Objects`. Make the `Objects` folder as needed.
+1. Open `{your-project-path}\Materials\Types\VegetationBending.materialtype`. Under `propertyLayout` > `propertyGroups`, you'll see there are many entries with `{your-path-to-o3de}`. Replace `{your-path-to-o3de}` with your appropriate path to the engine.
+   * For example, `C:/o3de/Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/BaseColorPropertyGroup.json`.
+   * Currently we cannot import property groups across gems, so we are hard coding the absolute path as a proof of concept, even though it is not portable. There is a GHI to enable importing across gems at [o3de#10623](https://github.com/o3de/o3de/issues/10623).
+1. Run the **Editor**, and the assets should automatically process. At this point, none of them should fail. If `VegetationBending.materialtype` fails to process, check that you used the correct paths in step 4.
+
+These template files were created by duplicating important parts of the `StandardPBR` files and then modifying them. When you create your own material types in the future, you can similalry duplicate `StandardPBR` files and work from there.
```

Comment:
This is really helpful information for those who want to develop their own material type. However, I don't think it fits in this section. At this point, the user is getting ready to do this tutorial, so information that's not specific to their current workflow at hand can be distracting and get forgotten. 

One way you might separate this information from the information relative to the tutorial is by putting it in a note callout at the end of the section. 
So, you can update to the following, and put it at the end of this section, (after "As a high-level overview..."):

```
{{< note > }}
These template files were created by duplicating important parts of the `StandardPBR` files and then modifying them. When you create your own material types in the future, you can similarly duplicate `StandardPBR` files and work from there.
{{< /note > }}
```